test/js/bun/test/expect.test.js

```diff
@@ -2743,7 +2836,6 @@ describe("expect()", () => {
     expect([]).toBeArrayOfSize(0);
     expect(new Array()).toBeArrayOfSize(0);
     expect([1, 2, 3, "🫓"]).toBeArrayOfSize(4);
-    expect((new Array() < string) | (number > (1, 2, 3, "🫓"))).toBeArrayOfSize(4);
```

Comment:
@paperdave I couldn't figure out what this test is supposed to look like... 🤔

Reply:
copied from test-test.test.ts. lol *what* the hell is that.

oh
this is supposed to be the TS code 
```ts
expect((new Array<string|number>(1, 2, 3, "🫓"))).toBeArrayOfSize(4)
```

i pasted it in a js file and it got reformatted as such

```ts
expect((new Array(1, 2, 3, "🫓"))).toBeArrayOfSize(4)
```

Reply:
Ah that makes sense &minus; many thanks 👍